app/upgrades/v1_5/upgrades.go

```diff
@@ -0,0 +1,119 @@
+	icacontrollertypes "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/controller/types"
+	icahosttypes "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/host/types"
+	ibctransfertypes "github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
+	"github.com/cosmos/ibc-go/v7/modules/core/exported"
+	ibctmmigrations "github.com/cosmos/ibc-go/v7/modules/light-clients/07-tendermint/migrations"
+
+	evmtypes "github.com/xpladev/ethermint/x/evm/types"
+	feemarkettypes "github.com/xpladev/ethermint/x/feemarket/types"
+
+	"github.com/xpladev/xpla/app/keepers"
+	rewardtypes "github.com/xpladev/xpla/x/reward/types"
+)
+
+func CreateUpgradeHandler(
+	mm *module.Manager,
+	configurator module.Configurator,
+	keepers *keepers.AppKeepers,
+	cdc codec.BinaryCodec,
+) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		// Set param key table for params module migration
+		for _, subspace := range keepers.ParamsKeeper.GetSubspaces() {
+			var keyTable paramstypes.KeyTable
+			switch subspace.Name() {
+			case authtypes.ModuleName:
+				keyTable = authtypes.ParamKeyTable() //nolint:staticcheck
+			case banktypes.ModuleName:
+				keyTable = banktypes.ParamKeyTable() //nolint:staticcheck,nolintlint
+			case stakingtypes.ModuleName:
+				keyTable = stakingtypes.ParamKeyTable()
+			case minttypes.ModuleName:
+				keyTable = minttypes.ParamKeyTable() //nolint:staticcheck
+			case distrtypes.ModuleName:
+				keyTable = distrtypes.ParamKeyTable() //nolint:staticcheck,nolintlint
+			case slashingtypes.ModuleName:
+				keyTable = slashingtypes.ParamKeyTable() //nolint:staticcheck
+			case govtypes.ModuleName:
+				keyTable = govv1.ParamKeyTable() //nolint:staticcheck
+			case crisistypes.ModuleName:
+				keyTable = crisistypes.ParamKeyTable() //nolint:staticcheck
+
+			// ibc
+			case ibctransfertypes.ModuleName:
+				keyTable = ibctransfertypes.ParamKeyTable()
+			case icacontrollertypes.SubModuleName:
+				keyTable = icacontrollertypes.ParamKeyTable()
+			case pfmroutertypes.ModuleName:
+				keyTable = pfmroutertypes.ParamKeyTable()
+			case icahosttypes.SubModuleName:
+				keyTable = icahosttypes.ParamKeyTable()
+
+			// ethermint
+			case evmtypes.ModuleName:
+				keyTable = evmtypes.ParamKeyTable() //nolint:staticcheck
+			case feemarkettypes.ModuleName:
+				keyTable = feemarkettypes.ParamKeyTable()
+
+			// wasm
+			case wasmtypes.ModuleName:
+				keyTable = wasmtypes.ParamKeyTable() //nolint:staticcheck
+
+			// xpla
+			case rewardtypes.ModuleName:
+				keyTable = rewardtypes.ParamKeyTable()
+
+			default:
+				continue
+			}
+			if !subspace.HasKeyTable() {
+				subspace.WithKeyTable(keyTable)
+			}
+		}
+
+		baseAppLegacySS := keepers.ParamsKeeper.Subspace(baseapp.Paramspace).WithKeyTable(paramstypes.ConsensusParamsKeyTable())
+		baseapp.MigrateParams(ctx, baseAppLegacySS, &keepers.ConsensusParamsKeeper)
+
```

Comment:
It should be set ConsensusParams in ctx. Because feekeeper abci need to consensusparams

Example) https://github.com/xpladev/ethermint/blob/d18ee5263447b4f96d1cc22de0c3ec3083fd4c3a/app/upgrades.go#L88-L89

Reply:
I fixed `ethermint/x/feemarket` for this, updated tag to rc2.